package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "czifui",
-  "version": "0.0.18",
+  "version": "0.0.25",
```

Comment:
I'm just curious, why did it go from 18 to 25?

Reply:
Great question! It's a long story lol

So Aspen uses happy path dev environment for local development now, which mounts the project directory in Docker local VM. BUT that doesn't mount the component library directory, so local project can't use the local component library

Thus, the only way for local project to use the component library is to publish it to `npm` once in a while 😆 

I try to decrease the publish frequency by testing things more thoroughly in Storybook. But sometimes I could brain fart and publish a new version, forgetting to export the new component first, thus needing to export, rebuild the package, bump the version, and publish again lol

I'm sure we can figure out a better process 😆